packages/website/ts/components/wallet/wallet_disconnected_item.tsx

```diff
@@ -0,0 +1,79 @@
+import { colors, Styles } from '@0xproject/react-shared';
+import FlatButton from 'material-ui/FlatButton';
+import ActionAccountBalanceWallet from 'material-ui/svg-icons/action/account-balance-wallet';
+import * as React from 'react';
+
+import { ProviderType } from 'ts/types';
+import { constants } from 'ts/utils/constants';
+
+export interface WalletDisconnectedItemProps {
+    providerType: ProviderType;
+    injectedProviderName: string;
+    onToggleLedgerDialog: () => void;
+}
+
+const styles: Styles = {
+    button: {
+        border: colors.walletBorder,
+        borderStyle: 'solid',
+        borderWidth: 1,
+        height: 80,
+    },
+    hrefAdjustment: {
+        paddingTop: 20, // For some reason when we set the href prop of a FlatButton material-ui reduces the top padding
```

Comment:
🤔

Reply:
Let's probably convert this to a `HACK` comment. Is there an outstanding Material-ui issue for this that we could link to?

Reply:
Couldn't find anything related in the issues or PRs, we are a good number of versions behind, we might want to consider upgrading material ui but not sure the refactor would be worth it. Plus the devs seems to be working towards a v1 release